testingutils/src/main/java/com/github/awsjavakit/testingutils/ApiGatewayRequestBuilder.java

```diff
@@ -83,4 +99,15 @@ public InputStream build() {
       throw new RuntimeException(e);
     }
   }
+
+  private Map<String, List<String>> convertToMultiValueHeaders(Map<String, String> headers) {
+    return headers.entrySet().stream()
+      .map(entry -> Map.entry(entry.getKey(), createSingleItemList(entry.getValue())))
```

Comment:
it seems that multivalue headers are not actually split in the event. We should investigate if this is a bug in `sam local  ` or this is how it is. For now splitting just puts all the values in a singleton list.